src/lib/eina/eina_debug_cpu.c

```diff
@@ -80,7 +80,10 @@ _sysmon(void *data EINA_UNUSED, Eina_Thread thr EINA_UNUSED)
              for (i = 0; i < _eina_debug_thread_active_num; i++)
                {
                   // get the correct clock id to use for the target thread
-                  pthread_getcpuclockid
+                  #ifndef _WIN32
+                     pthread_getcpuclockid(self, &cid);
+                  #endif
+               //   pthread_getcpuclockid
                      (_eina_debug_thread_active[i].thread, &cid);
```

Comment:
If `_WIN32` is defined, won't this generate...

```c
for (...)
  {
       (_eina_debug_thread_active[i].thread, &cid);

     ...
```

(aka a couble of dangling parameters in the middle of nowhere without a function call)?

Reply:
I noticed that this #ifndef is unnecessary. It will be removed in a next commit